### DISCUSSION TITLE
How to overwrite the font-family in G2

### DISCUSSION THREAD

**Discussion by aldo-sanchez** [antvis/G2#5499] [Q&A] [answered]
G2 version: `5.0.24`

I cannot find anywhere in the documentation where I can overwrite the font-family to an entire theme. For example, I want to use the `classic` theme with a different font-family. How can I set the font-family?

My main issue is that the tooltip font is appearing serif, because the default font-family is `Robot-Regular`. 

Notice the tooltip font:
![Screenshot 2023-08-31 at 11 55 33](https://github.com/antvis/G2/assets/22064588/529f787c-ed7e-4e0f-a967-6202cc2f318d)

I now manually change it in the browser inspector:
![Screenshot 2023-08-31 at 11 57 25](https://github.com/antvis/G2/assets/22064588/dcd7eead-e150-4efb-b940-4794cb9c6a12)

**Comment by hustcc** [comment, marked as answer]
You need to overwrite them by css, see the [demo](https://g2.antv.antgroup.com/examples/component/tooltip/#tooltip-style).

**Reply by aldo-sanchez** [reply]
Super helpful! thanks!
